extensions/hibernate-search-elasticsearch/deployment/src/main/java/io/quarkus/hibernate/search/elasticsearch/HibernateSearchLogFilterBuildStep.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.hibernate.search.elasticsearch;
+
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.logging.LogCleanupFilterBuildItem;
+
+/**
+ * Processor that sets up log filters for Hibernate Search
+ */
+public final class HibernateSearchLogFilterBuildStep {
+
+    @BuildStep
+    void setupLogFilters(BuildProducer<LogCleanupFilterBuildItem> filters) {
+        filters.produce(new LogCleanupFilterBuildItem("org.hibernate.search.engine.Version", "HSEARCH000034"));
```

Comment:
I wouldn't make it a separate class. We tend to centralize these things in the processor.

Reply:
Ok, I just copy-pasted from someplace else. Will do.